projects/storefrontapp-e2e-cypress/cypress/helpers/product-search-pricing-flow.ts

```diff
@@ -25,35 +26,41 @@ export function productPricingFlow() {
     '47 results for Digital Compacts'
   );
 
-  cy.get('cx-product-list-item').should(
+  cy.get(productItemSelector).should(
     'have.length',
     PRODUCT_LISTING.PRODUCTS_PER_PAGE
   );
 
-  checkFirstItem('DSC-S930');
+  cy.get(productNameSelector)
+    .first()
+    .invoke('text')
+    .should('match', /\w+/)
```

Comment:
I was thinking maybe putting the
cy.get(name)
.first
.invoke('text')
.should

inside a function in the helper and call it here since we're checking them 3 times

Reply:
It is being used 3 times, but one in a different way, which makes it only two. I'll refactor it anyway